commands.py

```diff
@@ -1004,22 +965,20 @@ def canvas_init(self):
         for c_handler_guild_id in self.bot.canvas_dict:
             guild = self.bot.guilds[[guild.id for guild in self.bot.guilds].index(int(c_handler_guild_id))]
 
-            if guild not in [ch.guild for ch in self.d_handler.canvas_handlers]:
+            if guild not in (ch.guild for ch in self.d_handler.canvas_handlers):
                 self.d_handler.canvas_handlers.append(CanvasHandler(CANVAS_API_URL, "", guild))
 
             c_handler = self._get_canvas_handler(guild)
             c_handler.track_course(tuple(self.bot.canvas_dict[c_handler_guild_id]["courses"]))
             live_channels_ids = self.bot.canvas_dict[c_handler_guild_id]["live_channels"]
-            live_channels = [channel for channel in guild.text_channels if channel.id in live_channels_ids]
```

Comment:
lol I personally prefer list comp over functional but that's just my taste

also I wonder whether there's a speed difference